cli/bin/setup

```diff
@@ -1,3 +1,14 @@
 #!/usr/bin/env bash
-echo "disabled until ZMQ is removed"
-# npm install --loglevel error --depth 0
+
+npm install --loglevel error --depth 0
+(cd ../exo-add && bin/setup && node_modules/.bin/build && echo 'exo-add √')
```

Comment:
Use [Morula](https://github.com/originate/morula) here so that when we add/rename/remove projects, this file still works?